client/dashboard/sites/notices.tsx

```diff
@@ -30,9 +30,15 @@ export const SitesNotices = () => {
 	const currentSearchParams = sitesRoute.useSearch();
 	const isRestoringAccount = !! currentSearchParams.restored;
 
-	if ( isRestoringAccount ) {
-		return (
-			<VStack className="sites-notices">
+	if ( ! window?.location?.pathname?.startsWith( '/v2' ) ) {
+		// Notices should not appear in the backported sites list.
+		return null;
+	}
```

Comment:
I changed the method we use for hiding the sites list notices, so that the padding would look the same across all the top-level pages. They should all use the `<PageLayout>` component's `notices` feature.